test/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -18,6 +18,9 @@ public class RepoDirectoriesProvider
         private string _corehostPackages;
         private string _corehostDummyPackages;
 
+        private string _targetRID;
+
+        public string TargetRID => _targetRID;
```

Comment:
Rather than adding this new property to `RepoDirectoriesProvider`, what do you think about simply changing TestProjectFixture to use `Environment.GetEnvironmentVariable("TEST_TARGETRID")` in place of `_sdkDotnet.GetRuntimeId()`? We already have multiple variables storing the RIDs, such as the `TARGETRID` and `TEST_TARGETRID` environment variables and the `TargetRID` and `ArtifactsTargetRID` BuildContext variables; if anything, it would be nice to consolidate these at some point rather than adding more.

Also, I think the name `TargetRID` could be confusing because it corresponds to the `TEST_TARGETRID` env var, not the `TARGETID` env var.

Additionally, I'm not sure that the `TargetRID` property really fits in this class anyway, because the rest of its properties represent directories.